src/Report/JunitReporter.php

```diff
@@ -34,6 +34,10 @@ public function getFormat()
      */
     public function generate(ReportSummary $reportSummary)
     {
+        if (!extension_loaded('dom')) {
```

Comment:
Can we check this when parsing the config? Than the user will know the report cannot be generated before fixing, maybe better UX?